linera-execution/src/runtime.rs

```diff
@@ -1385,6 +1424,33 @@ impl ServiceSyncRuntime {
         }
     }
 
+    pub(crate) fn preload_service(
+        &self,
+        id: UserApplicationId,
+        code: UserServiceCode,
+        description: UserApplicationDescription,
+    ) -> Result<(), ExecutionError> {
+        let this = self
+            .runtime
+            .0
+            .as_ref()
+            .expect("services shouldn't be preloaded while the runtime is being dropped");
+        let runtime_handle = this.clone();
+        let mut this_guard = this.inner();
+
+        if let std::collections::hash_map::Entry::Vacant(e) =
+            this_guard.loaded_applications.entry(id)
+        {
+            e.insert(LoadedApplication::new(
```

Comment:
Nit: avoiding usage of single letter bindings

```suggestion
        if let std::collections::hash_map::Entry::Vacant(entry) =
            this_guard.loaded_applications.entry(id)
        {
            entry.insert(LoadedApplication::new(
```